luxonis_train/strategies/base_strategy.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+
+import pytorch_lightning as pl
+from luxonis_ml.utils.registry import AutoRegisterMeta
+from torch.optim import Optimizer
+from torch.optim.lr_scheduler import LRScheduler
+
+from luxonis_train.utils.registry import STRATEGIES
+
+
+class BaseTrainingStrategy(
```

Comment:
Let's add this to "Customizations" docs as we now allow registering custom strategies